### PR TITLE
add documentation for using with Kubernetes

### PR DESCRIPTION
**Pull request by jberkhahn**

### CODE REVIEW

File: docs/kube/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: brokersecret
+type: Opaque
+data:
+  username: YnJva2Vy
+  password: cGFzc3dvcmQ=
```
Comment (swetharepakula) [inline docs/kube/secret.yaml]: Is this username and password hardcoded to this always? Or is this just a sample one for your deployment?

Comment (jberkhahn) [reply docs/kube/secret.yaml]: its assuming you're using the default username/password...like it says in the README.

Comment (nimakaviani) [reply docs/kube/secret.yaml]: username appears to be `base64` too. should be explicit in the doc.

Comment (nimakaviani) [reply docs/kube/secret.yaml]: never mind. comes in the next sentence.

Comment (nimakaviani) [reply docs/kube/secret.yaml]: `echo broker | base64` gives me `YnJva2VyCg==`

**Comment by nimakaviani** [comment]
Can we also collapse the 4 commits into 1

File: docs/kube/secret.yaml (hunk 1 above)
Comment (jberkhahn) [reply docs/kube/secret.yaml]: fixed

**Comment by jberkhahn** [comment]
squashed

**Comment by MHBauer** [comment]
I think we need to wait for the verification of bosh deployment success, and attempt to attach a similarly local minikube or kubeadm-dind deployed kubernetes to confirm this works.

**Comment by nimakaviani** [comment]
yeah I would like to try it against a kubernetes deployment. will give it a go tomorrow since I have a running deployment of the broker

**Pull request merged**